### PR TITLE
taxonomy: translate NOVA groups more broadly

### PR DESCRIPTION
**Pull request by teolemon**
### What
taxonomy: translate NOVA groups more broadly

### CODE REVIEW

**Comment by teolemon** [comment]
ERROR - is:meðhöndlun-matvæla already is a synonym of is:2-meðhöndlun-matvæla for entry en:2-processed-culinary-ingredients (nova_groups) - is:meðhöndlun-matvæla cannot be mapped to entry en:3-processed-foods / is:3-meðhöndlun-matvæla
[Fri Dec  8 13:04:34 2023] build_lang.pl: Errors in the nova_groups taxonomy definition at /opt/product-opener/lib/ProductOpener/Tags.pm line 1496.

File: taxonomies/nova_groups.txt [hunk 1]
```diff
@@ -41,14 +63,25 @@ ca:3 - Aliments processats, Aliments processats
 cs:3 - Zpracované potraviny
 de:3 - Verarbeitete Lebensmittel, Verarbeitete Lebensmittel
 es:3 - Alimentos procesados, Alimentos procesados
+eu:3 - Prozesatutako elikagaiak, Prozesatutako elikagaiak, 3
 fi:3 - Jalostetut ruoat, Jalostetut ruoat
 fr:3 - Aliments transformés, Aliments transformés
 he:3 - מזון מעובד
+hr:3 - Obrađena hrana, Obrađena hrana, 3
 hu:3 - Feldolgozott élelmiszerek
+is:3 - Meðhöndlun matvæla, Meðhöndlun matvæla, 3
```
Comment (teolemon) [inline taxonomies/nova_groups.txt:72]:
```suggestion
is:3 - Unnin matvæli, Unnin matvæli, 3
```

File: taxonomies/nova_groups.txt [hunk 2]
```diff
@@ -41,14 +63,25 @@ ca:3 - Aliments processats, Aliments processats
...
 it:3 - Alimenti lavorati
+lt:3 - Apdoroti maisto produktai, Apdoroti maisto produktai, 3
+lv:3 - Apstrādāti pārtikas produkti, Apstrādāti pārtikas produkti, 3
+mt:3 - Ikel imxerrda, Ikel imxerrda, 3
```
Comment (teolemon) [inline taxonomies/nova_groups.txt:76]:
```suggestion
```

File: taxonomies/nova_groups.txt [hunk 3]
```diff
@@ -58,14 +90,24 @@ ca:4 - Aliments i begudes ultraprocessats, Aliments i begudes ultraprocessats
...
 hu:4 - Többszörösen feldolgozott élelmiszer- és italtermékek
+is:4 - Óþarfar matvörur og drykkir, Óþarfar matvörur og drykkir, 4
 it:4 - Alimenti e bevande ultra lavorati
+lt:4 - Labai apdoroti maisto produktai ir gėrimai, Labai apdoroti maisto produktai ir gėrimai, 4
+lv:4 - Ļoti apstrādāti pārtikas produkti un dzērieni, Ļoti apstrādāti pārtikas produkti un dzērieni, 4
+mt:4 - Prodotti tal-ikel u xorb ultra imxerrda, Prodotti tal-ikel u xorb ultra imxerrda, 4
 nl:4 - Ultrabewerkte dranken en levensmiddelen
+no:4 - Ultra bearbeidede mat- og drikkevarer, Ultra bearbeidede mat- og drikkevarer, 4
 pl:4 - Żywność wysoko przetworzona
 pt:3 - Produtos alimentares e bebidas ultra-processados, Produtos alimentares e bebidas ultra-processados
```
Comment (benbenben2) [inline taxonomies/nova_groups.txt:107]: Not related to your contribution, but it seems that there was a typo there

```suggestion
pt:4 - Produtos alimentares e bebidas ultra-processados, Produtos alimentares e bebidas ultra-processados
```

**Review by benbenben2 — APPROVED** [review]
Looks good to me.

**Review by stephanegigandet — APPROVED** [review]

**Pull request merged**